idseq_dag/steps/run_star.py

```diff
@@ -71,6 +84,13 @@ def __init__(self, *args):
         super().__init__(*args)
         self.sequence_input_files = None
         self.validated_input_counts_file = None
+        if (len(self.output_files_local()) == 3):
+            self.bam_output = self.output_files_local()[2]
```

Comment:
> Also since we don't want to do this for single reads, the output changes depending on single or pair reads. 

The code looks like the reverse logic. Paired end reads are assumed if the bam file is in the dag json file. I guess you are adding the output file dynamically from idseq web.